payments/schema/mutations.py

```diff
@@ -416,6 +416,28 @@ def mutate_and_get_payload(cls, root, info, **input):
         return ConfirmPaymentMutation(url=payment_url)
 
 
+class CancelOrderMutation(graphene.ClientIDMutation):
+    class Input:
+        order_number = graphene.String(required=True)
+
+    @classmethod
+    def mutate_and_get_payload(cls, root, info, **input):
```

Comment:
Should this be decorated with `@change_permission_required(Order)`?

Reply:
It's a public endpoint, so we can't check if the user has permissions.

Reply:
Ooh right, mixed this with the discussion about cancelling sent out orders from the admin side :)